interface/backend/images/tests.py

```diff
@@ -14,13 +20,163 @@ def test_get_create_image_series(self):
+        self.assertEqual(len(cm.output), 1)
+        self.assertIn('RescaleSlope', cm.output[0])
+        self.assertIn(path, cm.output[0])
+
+        # reset for other tests
+        ImageFile.DICOM_PROPERTIES.update({
+            'RescaleSlope': lambda x: {'rescale_slope': int(x)},
+        })
+
+    def test_parsed_property_no_model_attribute(self):
+        uri = '/images/LIDC-IDRI-0001/' \
+              '1.3.6.1.4.1.14519.5.2.1.6279.6001.298806137288633453246975630178/' \
+              '1.3.6.1.4.1.14519.5.2.1.6279.6001.179049373636438705059720603192'
+
+        path = os.path.join(uri, '-132.500000.dcm')
+
+        image_series, created = ImageSeries.get_or_create(uri)
+
+        # CASE ONE, Dicom prop does not exist, attr is not field
+        ImageFile.DICOM_PROPERTIES.update({
+            'DoesNotExist1': 'no_attribute',
+        })
+
+        # expected behavior is to log a warning
+        with self.assertLogs(level='WARN') as cm:
+            ImageFile(path=path, series=image_series).save()
+
+        self.assertEqual(len(cm.output), 1)
+        self.assertIn("'no_attribute' not a field", cm.output[0])
+
+        # reset
+        ImageFile.DICOM_PROPERTIES.pop('DoesNotExist1')
+
+        # CASE TWO, Dicom prop exists, attr is not field
+        ImageFile.DICOM_PROPERTIES.update({
+            'RescaleSlope': 'no_attribute2',
+        })
+
+        # expected behavior is to log a warning
+        with self.assertLogs(level='WARN') as cm:
+            ImageFile(path=path, series=image_series).save()
+
+        self.assertEqual(len(cm.output), 1)
+        self.assertIn("'no_attribute2' not a field", cm.output[0])
+
+        ImageFile.DICOM_PROPERTIES.update({
+            'RescaleSlope': lambda x: {'rescale_slope': int(x)},
+        })
+
+        # CASE THREE, Dicom prop does not exist, field does exist
+        ImageFile.DICOM_PROPERTIES.pop('Rows')
+        ImageFile.DICOM_PROPERTIES.update({
+            'DoesNotExist2': 'rows',
+        })
+
+        # no warnings should be logged
+        try:
+            with self.assertLogs(level='WARN') as cm:
+                img_file = ImageFile(path=path, series=image_series)
+                img_file.save()
+        except AssertionError as e:
+            print(str(e))
+            self.assertEqual(str(e), "no logs of level WARNING or higher triggered on root")
+
+        self.assertIsNone(img_file.rows)
+
+        # reset
+        ImageFile.DICOM_PROPERTIES['Rows'] = 'rows'
+
+        # CASE FOUR, Dicom prop does not exist, rows set multiple times
+        ImageFile.DICOM_PROPERTIES.update({
+            'DoesNotExist2': 'rows',
+        })
+
+        # expected behavior is to log a warning
+        with self.assertLogs(level='WARN') as cm:
+            ImageFile(path=path, series=image_series).save()
+
+        self.assertEqual(len(cm.output), 1)
+        self.assertIn("Tried to set 'rows' more than once", cm.output[0])
+
+        ImageFile.DICOM_PROPERTIES.pop('DoesNotExist2')
```

Comment:
Can `DICOM_PROPERTIES` updates be wrapped in setup/cleanup steps so that the manual resetting won't be needed?